src/components/typography/typography.tsx

```diff
@@ -166,3 +166,26 @@ export const TextLink = styled(Link)<{ small?: boolean }>`
     text-decoration: underline;
   }
 `;
+
+export const LeadBoxTitle = styled.p`
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 110%;
+  margin: 0 auto 24px;
+  max-width: 320px;
+`;
+
+export const LeadboxSubtitle = styled.p`
+  font-size: 14px;
+  line-height: 150%;
+  font-weight: bold;
+  margin: 0 auto;
+  max-width: 320px;
+`;
+
+export const LeadboxText = styled.p`
+  font-size: 14px;
+  line-height: 150%;
+  margin: 0 auto;
+  max-width: 320px;
+`;
```

Comment:
I was wondering if we could reuse existing typography settings here instead of introducing new ones. These last two ones look very similar to `ImageCardTitle` and `ImageCardSubtitle`. Maybe we can combine some of the existing ones so we don't have to deal with so many different typography options? I remember Eric had some overview over all the used type faces, maybe we can ask him to receive a definitive list of everything that is used on the website and clean this file up.